Tests/KituraSampleRouterTests/KituraTest.swift

```diff
@@ -41,18 +41,22 @@ extension KituraTest {
 
     func performServerTest(asyncTasks: @escaping (XCTestExpectation) -> Void...) {
         let router = RouterCreator.create()
-        Kitura.addHTTPServer(onPort: 8090, with: router)
-        Kitura.start()
-        sleep(1)
```

Comment:
@na-gupta So should [KituraTest in Kitura](https://github.com/IBM-Swift/Kitura/blob/master/Tests/KituraTests/KituraTest.swift#L52) not use `started` as well?